design/modular-development.md

```diff
@@ -264,15 +264,20 @@ To support this in our packages, we should think about how we support a hybrid p
 
 ### Hybrid Design
 
-For design considerations and code re-use, we could deploy a hybrid solution to this problem by using the single method exports and still expose the existing `ServiceClient` which simply acts as a proxy to the the underlying method.  
+For design considerations and code re-use, we could deploy a hybrid solution to this problem by using the single method exports and still expose the existing `ServiceClient` which simply acts as a proxy to the underlying method.  
 
 ```typescript
-import { NotificationHubsClientContext, createClientContext, createRegistrationId } from "@azure/notification-hubs/api";
+import { 
```

Comment:
I prefer the existing formatting - could we revert this change?

Reply:
OK.